distributed/hdfs.py

```diff
@@ -21,7 +22,7 @@
 
 
 def walk_glob(hdfs, path):
-    if '*' not in path and hdfs.info(path)['kind'] == 'directory':
+    if '*' not in path and hdfs.info(path).get('kind', '') == 'directory':
```

Comment:
To be pedantic here, in an ideal world no source change should really ever happen without a similar change in tests.  Is it ok if I revert this change?  If not why not?